### PR TITLE
Save fields on change

### PR DESCRIPTION
**Pull request by elenatorro**
Related with https://github.com/CartoDB/carto-vl/issues/734 and https://github.com/CartoDB/carto-vl/pull/735

### CODE REVIEW

File: examples/editor/index.js [hunk 1]
```diff
@@ -255,6 +263,8 @@ const superRefresh = (opts) => {
     const vizStr = document.getElementById('styleEntry').value;
     const viz = new carto.Viz(vizStr);
 
+    saveConfig();
```
Comment (Jesus89) [inline examples/editor/index.js]: Why is this added?

Comment (elenatorro) [reply examples/editor/index.js]: I added it because currently, every time there is an error, if you change the fields after that those changes are not saved. There may be a better way to do so, actually. Any suggestion?

Comment (Jesus89) [reply examples/editor/index.js]: OK. I would move the function to the top to save after Source or Viz errors. What do you think?

Comment (elenatorro) [reply examples/editor/index.js]: 👍 Agree!
